config/development.dist.yml

```diff
@@ -17,9 +17,6 @@ application:
   show_contrib_banner: true
   venue_image_path: /assets/img/venue.jpg
 
-cache:
-  enabled: false
```

Comment:
Symfony needs a cache for its container and router. Because of this, I removed to option to disable the cache.